README.md

```diff
@@ -1,6 +1,61 @@
 # slacker
-Slack REPL with handy utilities
+Slacker is a tool designed to make it easier to do admin tasks and general utility for several workspaces (teams). It supports a REPL for inputting and evaluating commands, along with a CLI for running a single command.
 
+# Table of Contents
+* [First Run](#first-run)
+* [General Commands](#general-commands)
+* [CLI Usage](#cli-usage)
+* [Development Environment](#development-environment)
+* [Contributing](#contributing)
 
-# Development environment
-Use `virtualenv -p python3 .venv` from the project root directory to setup the development environment. To activate the environment type `source .venv/bin/activate`. Next, run `pip install -r requirements.txt` to get the required dependencies. Or run `make setup`.
+# First Run
+The first invocation of Slacker requires to setup a workspace using the `--init` argument:
```

Comment:
Wondering if we should add a note that here that recommends that users should use the `user` api token instead of bot?

Reply:
That's a good point. I will add this.